volttrontesting/testutils/test_base_historian_unit.py

```diff
@@ -36,19 +36,18 @@ def test_base_historian_agent_should_filter_duplicates(base_historian_agent):
             message=f"last_duplicate_{num}",
         )
 
-    # Add unique records to queue
-    for num in range(2, 5):
-        base_historian_agent._capture_record_data(
-            peer=None,
-            sender=None,
-            bus=None,
-            topic=f"unique_record_topic{num}",
-            headers={
-                "Date": f"2020-11-17 21:2{num}:10.189393+00:00",
-                "TimeStamp": f"2020-11-17 21:2{num}:10.189393+00:00",
-            },
-            message=f"unique_record_{num}",
-        )
+    # Add a unique record to queue
+    base_historian_agent._capture_record_data(
```

Comment:
Are you sure we want tor remove the f-string here?

Reply:
I updated the test to only add one unique item rather than multiple so the expected_to_publish_list wasn't too long, just so the test was a little easier to follow.

Reply:
The thing is we generally see a large number of duplicates when we have them so we should probably test for many of them not just single...the timestamp resolution should still only give us one.

Reply:
We are testing for multiple duplicates. The first for loop adds three duplicate items. Then we add one unique item after that.